docs/index.md

```diff
@@ -161,7 +164,7 @@ $getInitials = f\compose(f\join(' '), f\map(f\compose(f\concat_right('.'), f\pro
 $getInitials('Miles Davis'); // M. D.
 ```
 
-If you start at the right side you can follow along with the path your arguments will travel through 
+If you start at the right side you can follow along with the path your arguments will travel through
```

Comment:
Can you try and return the whitespace on these lines?  
Two or more spaces at the end of a line in Markdown translate to `<br>` instead of a new `<p>`, so these spaces are intentional. 

For what it's worth, I've configured my editor to strip whitespace as well, but not on Markdown files to prevent this.